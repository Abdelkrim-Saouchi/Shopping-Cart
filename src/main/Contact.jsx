import fbIcon from '../assets/facebook.png';
import twitterIcon from '../assets/twitter.png';
import githubIcon from '../assets/github.png';

const Contact = () => {
  return (
    <main className="contacts container">
      <div className="social-links">
        <a href="https://www.facebook.com/">
          <img src={fbIcon} alt="facebook icon" />
        </a>
        <a href="https://twitter.com/">
          <img src={twitterIcon} alt="twitter icon" />
        </a>
        <a href="https://github.com/">
          <img src={githubIcon} alt="github icon" />
        </a>
      </div>
      <div className="address">
        <h3>1044 Street, Bordj Bou Arreridj. Algeria</h3>
      </div>
    </main>
  );
};

export default Contact;
