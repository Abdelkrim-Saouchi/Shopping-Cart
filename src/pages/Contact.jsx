import fbIcon from '../assets/facebook.png';
import githubIcon from '../assets/github.png';
import twitterIcon from '../assets/twitter.png';
import styles from './Contact.module.css';

const Contact = () => {
  return (
    <main className={'container ' + styles.contacts}>
      <div className={styles.socialLinks}>
        <a href="https://www.facebook.com/">
          <img src={fbIcon} alt="facebook" />
        </a>
        <a href="https://twitter.com/">
          <img src={twitterIcon} alt="twitter" />
        </a>
        <a href="https://github.com/">
          <img src={githubIcon} alt="github" />
        </a>
      </div>
      <div>
        <h3>1044 Street, Bordj Bou Arreridj. Algeria</h3>
      </div>
    </main>
  );
};

export default Contact;
