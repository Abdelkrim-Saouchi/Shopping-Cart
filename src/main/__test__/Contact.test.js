import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import { BrowserRouter } from 'react-router-dom';
import Contact from '../Contact';

test('renders homepage correctly', () => {
  const { container } = render(<Contact />, { wrapper: BrowserRouter });
  expect(container).toMatchSnapshot();
});
