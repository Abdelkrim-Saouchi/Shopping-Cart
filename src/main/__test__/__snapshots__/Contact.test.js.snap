// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage correctly 1`] = `
<div>
  <main
    class="contacts container"
  >
    <div
      class="social-links"
    >
      <a
        href="https://www.facebook.com/"
      >
        <img
          alt="facebook icon"
          src="facebook.png"
        />
      </a>
      <a
        href="https://twitter.com/"
      >
        <img
          alt="twitter icon"
          src="twitter.png"
        />
      </a>
      <a
        href="https://github.com/"
      >
        <img
          alt="github icon"
          src="github.png"
        />
      </a>
    </div>
    <div
      class="address"
    >
      <h3>
        1044 Street, Bordj Bou Arreridj. Algeria
      </h3>
    </div>
  </main>
</div>
`;
